sklearn_numba_dpex/kmeans/drivers.py

```diff
@@ -356,7 +346,7 @@ def __call__(
             reset_centroids_private_copies(new_centroids_t_private_copies)
 
             # TODO: implement special case where only one copy is needed
-            fixed_window_fused_kernel(
+            fused_lloyd_fixed_window_single_step(
```

Comment:
If the other suggestion is approved, this one also must.
```suggestion
            fused_lloyd_fixed_window_single_step_kernel(
```